src/main/java/com/devteam/sandboxbasedapp/service/ParkingSpaceService.java

```diff
@@ -0,0 +1,9 @@
+package com.devteam.sandboxbasedapp.service;
+
+import com.devteam.sandboxbasedapp.controller.dto.ParkingSpaceDTO;
+
+import java.util.List;
+
+public interface ParkingSpaceService {
```

Comment:
I don't see any advantages of using this interface. Let's get rid of it and rename TransactionalParkingSpaceService to ParkingSpaceService

Reply:
Done.